apps/docs/pages/guides/functions/debugging.mdx

```diff
@@ -11,82 +11,135 @@ export const meta = {
 
-## Limitations
+### How to use logs
 
-- Outgoing connections to ports `25`, `465`, and `587` are not allowed.
-- Serving of HTML content is not supported (`GET` requests that return `text/html` will be rewritten to `text/plain`).
-- Deno FileSystem APIs are not available to use within Edge Functions.
-- Importing modules using `npm:` specifier is currently not supported ([use a CDN to load npm modules](/docs/guides/functions/import-maps)).
+- **Uncaught exceptions**: Uncaught exceptions thrown by a function during execution will be automatically logged, allowing you to view the error message and stack trace in the Logs tool.
+- **Custom log events**: You can also use `console.log`, `console.error`, and `console.warn` in your code to emit custom log events. These events will also appear in the Logs tool. For example, you can use `console.log` to print the value of a variable, or `console.error` to report an unexpected condition.
+- **Message length and event limits**: A custom log message can contain up to 10,000 characters. A function can log up to 100 events within a 10 second period.
 
-## Troubleshooting
+Here is an example of how to use `console.log` in your function:
 
-### Unable to call Edge Function
+```
+Deno.serve(async (_req) => {
+  const { name } = await req.json()
+  const data = {
+    message: `Hello ${name}!`,
+  }
 
-If you're unable to call your Edge Function or are experiencing any CORS issues:
+  console.log(`Name: ${name}`)
 
-- Make sure you followed the [CORS guide](/docs/guides/functions/cors).
-- Check your function logs. Navigate to the [Functions section in your dashboard](https://supabase.com/dashboard/project/_/functions), select your function from the list, and click `Logs`. Check for any errors.
+  return new Response(JSON.stringify(data), { headers: { 'Content-Type': 'application/json' } })
+})
+```
 
-There are two debugging tools available: Invocations and Logs. Invocations shows the Request and Response for each execution, while Logs shows any platform events, including deployments and errors.
+## Troubleshooting
+
+If you encounter any problems or issues with your Edge Functions, here are some tips and steps to help you resolve them.
 
 ### Unable to deploy Edge Function
 
-- Make sure you're on the latest version of the [Supabase CLI](/docs/guides/cli#updates).
-- Run the deploy command with the `--debug` flag.
-- Run the deploy command with the `--legacy-bundle` flag.
+- Make sure you're on the latest version of the [Supabase CLI](/docs/guides/cli#updates).
+- Run the deploy command with the `-debug` flag. This flag will show more details and information about the deployment process and any errors that may occur.
+- Run the deploy command with the `-legacy-bundle` flag. This flag will use a different bundling method that may work better for some functions.
```

Comment:
This no loner works. we can remove this option from docs.

Reply:
You mean the commands with the `-debug` and `--legacy-bundle` flag?